deploy/docker/tests/pg-test-utils.sh

```
@@ -0,0 +1,65 @@
+# Function to check if the Appsmith instance is up
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)